kubescaler/cluster.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         min_nodes=0,
         machine_type=None,
         kubernetes_version=None,
+        eks_nodegroup=0
```

Comment:
This should be a boolean that has a default (set to what you think is best, and that should e the same value in the calling scripts). I'm guessing for 0 this should be False.